spinn_front_end_common/utility_models/data_speed_up_packet_gatherer_machine_vertex.py

```diff
@@ -358,7 +372,7 @@ def _determine_and_retransmit_missing_seq_nums(
         # locate missing sequence numbers from pile
         missing_seq_nums = self._calculate_missing_seq_nums(seq_nums)
         lost_seq_nums.append(len(missing_seq_nums))
-        # self._print_missing(seq_nums)
+        self._print_missing(seq_nums)
```

Comment:
Should this be disabled for the live code?

Reply:
yes, was used for debug with all the faff of the last few days.